modules/taxonomy/log_category/src/Form/LogCategorizeActionForm.php

```diff
@@ -155,6 +155,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['category'] = [
       '#type' => 'select',
       '#title' => $this->t('Log category'),
+      '#description' => $this->t('Use this to organize your logs into categories for easier searching and filtering later.'),
```

Comment:
Could we condense this? It seems a bit verbose and inconsistent from other descriptions, maybe don't include "your"?

"Organize logs into categories for easier searching and filtering."

This is also nearly identical to the flags description. Can we emphasize that you can build your own log category hierarchy? In Rothamsted modules we have included a link to the hierarchy, but not sure how viable that may be here.

Reply:
These feel like bigger considerations than just consistency. Not that I disagree - but maybe we can tackle them in a different PR?

> In Rothamsted modules we have included a link to the hierarchy, but not sure how viable that may be here.

This would probably need an access check too - not everyone who can edit fields can access the hierarchy, I think.